src/ReactiveDomain/ReactiveDomain.Tests/Logging/with_message_logging_enabled.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using EventStore.ClientAPI;
+using ReactiveDomain.Logging;
+
+namespace ReactiveDomain.Tests.Logging
+{
+    // ReSharper disable once InconsistentNaming
+    public abstract class with_message_logging_enabled :
+        with_event_store
+    {
+        protected EventStoreMessageLogger Logging;
+        protected string StreamName = $"Test-{Guid.NewGuid():N}-";
+
+        protected override void Given()
+        {
+            base.Given();
+
+            // instantiate Logger class that inherits from QueuedSubscriber
+            Logging = new EventStoreMessageLogger(Bus,
+                EventStore.Connection,
+                StreamName,
+                true);
+
+            Thread.Sleep(2000); // needs a bit of time to set up the ES
+        }
+
+        #region IDisposable
+
+        private bool _disposed;
+
+        protected override void Dispose(bool disposing)
+        {
+            if (_disposed) return;
+            if (disposing)
+            {
+                Logging?.Dispose();
+                //TODO: figure out how to delete test logging streams
```

Comment:
I think the entire ES repo should be deleted.

Reply:
tricky. we are running a batch of tests. I wouldn't want to delete the repo out from under other tests.
The stream has a unique name, but at least right now the ES doesn't.  Maybe I can figure out whether it's possible to have the same ES exe managing multiple stores. I haven't looked that far into it.